src/cli-command/import-command.ts

```diff
@@ -2,19 +2,74 @@ import TSVFileReader from '../common/file-reader/tsv-file-reader.js';
 import { CliCommandInterface } from './cli-command.interface.js';
 import { createFilm, getErrorMessage } from '../utils/common.js';
 
+import DatabaseService from '../common/database-client/database.service.js';
+import ConsoleLoggerService from '../common/logger/console-logger.service.js';
+import { getURI } from '../utils/db.js';
+import { UserServiceInterface } from '../modules/user/user-service.interface.js';
+import { FilmServiceInterface } from '../modules/film/film-service.interface.js';
+import UserService from '../modules/user/user.service.js';
+import FilmService from '../modules/film/film.service.js';
+import { FilmModel } from '../modules/film/film.entity.js';
+import { UserModel } from '../modules/user/user.entity.js';
+import { Film } from '../types/film.type.js';
+import { LoggerInterface } from '../common/logger/logger.interface.js';
+import { DatabaseInterface } from '../common/database-client/database.interface.js';
+
+const DEFAULT_DB_PORT = 27017;
+const DEFAULT_USER_PASSWORD = '123456';
+
 export default class ImportCommand implements CliCommandInterface {
   public readonly name = '--import';
 
-  private onLine(line: string) {
+
+  private userService!: UserServiceInterface;
```

Comment:
```suggestion
  private userService: UserServiceInterface;
```
восклицательный знак нужен только если ты не объявляешь что-то в конструкторе